client/components/review-field.js

```diff
@@ -56,6 +56,17 @@ class ReviewField extends React.Component {
       )
     }
 
+    if ((this.props.legacyGranted || this.props.isGranted) && (this.props.name === 'continuation' && !isUndefined(value)))  {
```

Comment:
Should the logic here around being a granted version sit outside this component?

As it stands, if it is not a granted version then the component will render `<p>No answer provided</p>` which feels incorrect to me.